src/tag/index.js

```diff
@@ -13,9 +13,9 @@ class Tag extends PureComponent {
     onDelete: PropTypes.func
   }
 
-  handleClick = () => {
+  handleClick = e => {
     const { id, onDelete } = this.props
-
+    e.stopPropagation()
```

Comment:
Don't remember why but I think we also need to add `e.stopImmediatePropagation()` too.

Reply:
Right. We can use `stopImmediatePropagation()` because it implicitly calls `stopPropagation()`. I'll update the code.